doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -579,6 +579,31 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
    :start-after: fota_upgrades_zigbee_start
    :end-before: fota_upgrades_zigbee_end
 
+.. _ug_nrf5340_image_slot_numbering:
+
+Image IDs and slot numbering
+****************************
+
+Each image in the DFU (Device Firmware Update) process on the nRF5340 DK is identified by a unique ID, corresponding to a specific memory slot for each core.
+This configuration typically follows:
+
++--------+------------------------------------------------------------+-----------+-------+
+| Slot   | Description                                                | DTS label | n     |
```

Comment:
Please see original ticket's comments/discussion [here](https://nordicsemi.atlassian.net/browse/NCSDK-23134)
And, verify if information in this table is correct?